src/Classes/ServiceAPI/MyRadio_Podcast.php

```diff
@@ -855,13 +856,27 @@ public function convert()
      *
      * @param int $num_results The number of results to return per page. 0 for all podcasts.
      * @param int $page The page required.
-     * @param bool $includeSuspended Whether to include suspended podcasts in the result
+     * @param bool $include_suspended Whether to include suspended podcasts in the result
+     * @param bool $include_pending Whether to include pending (future publish/processing) podcasts in the result
      *
      * @return Array[MyRadio_Podcast]
      */
-    public static function getAllPodcasts($num_results = 0, $page = 1, $includeSuspended = true)
+    public static function getAllPodcasts($num_results = 0, $page = 1, $include_suspended = true, $include_pending = true)
     {
-        $where = !$includeSuspended ? 'WHERE suspended = false ': '';
+        $where = '';
+        if (!$include_suspended || !$include_pending) {
+            $where = 'WHERE ';
+            if (!$include_suspended) {
+                $where .= 'suspended = false';
+            }
+            if (!$include_suspended && !$include_pending) {
+                $where .= ' AND ';
+            }
+            if (!$include_pending) {
+                $where .= 'NOT submitted IS NULL';
```

Comment:
`submitted IS NOT NULL` reads a bit cleaner, IMHO

Reply:
Mmm, yes, I think this was a testing leftover from a mistake of mine.